blog/fine-grained-news-2024-11.md

```diff
@@ -0,0 +1,107 @@
+
+![OpenFGA team group photo in Chicago](../static/img/blog/fgn-2024-11-chicago-offsite-team-photo.png)
+
+*The OpenFGA team in Chicago*
+
+### **New Modeling Demos Available!**
+Learn how to model fine-grained authorization in OpenFGA's domain-specific language step-by-step with our [new demo video series](https://www.youtube.com/playlist?list=PLUR5l-oTFZqWaDdhEOVt_IfPOIbKo1Ypt)! Starting with the basics and gradually adding complexity, this playlist is your guide to mastering OpenFGA modeling.
+
+### **Monthly Community Meeting**
+
+Join our vibin-depth monthly community discussions every second Thursday at [11 AM Eastern Time (US)](https://www.worldtimebuddy.com/?qm=1&lid=12,100,5,8&h=5&sln=11-12&hf=1). Check out our [meeting details](https://github.com/openfga/community/blob/main/community-meetings.md#:~:text=OpenFGA%20Community%20Meetings) for more information.
+
+November's highlights included:
+- [Sebastian Döll](https://www.linkedin.com/in/katallaxie/) from ZEISS showcasing their Terraform/OpenFGA integration.
+- [Justin Cohen](https://www.linkedin.com/in/justincoh/) demonstrating the new Batch Check functionality.
+
+**Join us Thursday, December 12, 2024** for our next meeting. Can't make it? Catch up on our [latest recording](https://youtu.be/4MGF4rTzhbA?si=iGcoZTw8T99E0LKs) or browse previous sessions on our [YouTube channel](https://www.youtube.com/@OpenFGA).
+
+### **Blogs and Videos for AuthZ Fans:**  
+ - **Granting TTL based permissions in OpenFGA:** Implement TTL-based permissions in OpenFGA for time-limited access control. [Read more on Medium...](https://medium.com/@shruti1810/granting-ttl-based-permissions-in-openfga-2ed2073931c3)
+
+- **Overcoming Security Challenges in Protecting Shared Generative AI Environments:** Explore solutions for ensuring secure, scalable, and efficient multi-tenancy in generative AI environments. [Read more on Medium...](https://towardsdatascience.com/overcoming-security-challenges-in-protecting-shared-generative-ai-environments-1ffb27da1bde)
+
+- **Fine-Grained Authorization for Backstage using OpenFGA:** Learn how OpenFGA enables dynamic fine-grained authorization in Backstage through ReBAC models and seamless policy updates. [See the webinar on YouTube...](https://www.youtube.com/watch?v=wWFbLPvwOyQ)
+
+## **New Adopters**
+
+- Are you using OpenFGA in production? Join our growing community of adopters! Add your company to our [ADOPTERS.md](https://github.com/openfga/community/blob/main/ADOPTERS.md#companiesprojects-using-openfga-in-production) file with a quick PR.
+
+- Do you offer OpenFGA implementation services? Get listed in our [Implementation Services](https://github.com/openfga/community/blob/main/ADOPTERS.md#companies-offering-openfga-implementation-services) directory. Note: Listings are community-contributed and not officially endorsed by the OpenFGA project.
+
+## **Announcements**
+
+### **OpenFGA Rankked #5 in CNCF Project Contributions!**
```

Comment:
```suggestion
### **OpenFGA Ranked #5 in CNCF Project Contributions!**
```

Reply:
This line has previously been updated to correct a formatting error, so this suggested change may be referencing a change that has already been made.